Documentation/Global/Image.rst

```diff
@@ -9,73 +9,148 @@ Image ViewHelper `<f:image>`
+folder of an extension.
+
+If you installed the extension via Composer **before** it had a folder of that
+exact name, reinstall it so that the symlinks in folder :path:`public/_assets`
+are correctly created for you.
+
+..  note::
+
+    Avoid using image operations, such as the arguments
+    :ref:`width <t3viewhelper:viewhelper-argument-typo3-cms-fluid-viewhelpers-imageviewhelper-width>` or
+    :ref:`crop <t3viewhelper:viewhelper-argument-typo3-cms-fluid-viewhelpers-imageviewhelper-crop>`
+    with images included from an extensions' assets. Each
+    non-FAL image operation creates a "fake" FAL record, which may lead to problems.
+
+..  _typo3-fluid-image-image:
+
+Argument image - display images from the folder fileadmin / from FAL
+====================================================================
+
+Folders from within the folder :path:`fileadmin` are usually accessed using the
+`file abstraction layer (FAL) <https://docs.typo3.org/permalink/t3coreapi:fal-introduction>`_.
 
-Examples
-========
+For example, if you want to display an image that is attached to a content
+element, the content element typically stores a reference count of attached images in
+its field `media`. Intermediate tables are then used to attach the actual image
+to the content elements. This allows backend users to move or rename files,
+while the files connection is not lost in the content element.
 
-Default
--------
+..  _typo3-fluid-image-fluidtemplate:
 
-::
+FLUIDTEMPLATE and data processors and images
+--------------------------------------------
 
-   <f:image src="EXT:myext/Resources/Public/typo3_logo.png" alt="alt text" />
+In a TypoScript `FLUIDTEMPLATE <https://docs.typo3.org/permalink/t3tsref:cobj-template>`_
+you can use the `files data processor <https://docs.typo3.org/permalink/t3tsref:filesprocessor>`_
+to display an image with the `<f:image>` ViewHelper.
 
-Output in frontend::
+See `Example: Render the images stored in field image <https://docs.typo3.org/permalink/t3tsref:filesprocessor-example-render-image>`_
+in the TypoScript reference.
 
-   <img alt="alt text" src="typo3conf/ext/myext/Resources/Public/typo3_logo.png" width="396" height="375" />
+You can then use the :ref:`image <t3viewhelper:viewhelper-argument-typo3-cms-fluid-viewhelpers-imageviewhelper-image>`
+argument to pass the :php-short:`\TYPO3\CMS\Core\Resource\File` or
+:php-short:`\TYPO3\CMS\Core\Resource\FileReference`
+provided by the TypoScript data processor to the ViewHelper.
 
-or in backend::
+..  code-block:: html
+
+    <f:for each="{images}" as="image">
+        <f:image image="{image}" class="something" height="250" alt="Description"/>
+    </f:for>
 
-   <img alt="alt text" src="../typo3conf/ext/viewhelpertest/Resources/Public/typo3_logo.png" width="396" height="375" />
+The :abbr:`FAL (file abstraction layer)` will take care of scaling the image and
+converting it to a web format if necessary (for example a pdf to a png).
```

Comment:
Maybe you mean "jpg" instead of "pdf", right? Never heard that images can be converted to pdf

Reply:
a pdf file can be converted into a png

Reply:
I have read it wrong. I have readed `for example a pdf OR a png` instead of `for example a pdf TO a png`. Sorry.